src/dbnode/environment/config.go

```diff
@@ -109,11 +109,10 @@ type ConfigureResults struct {
 
 // ConfigurationParameters are options used to create new ConfigureResults
 type ConfigurationParameters struct {
-	InstrumentOpts             instrument.Options
-	HashingSeed                uint32
-	HostID                     string
-	NamespaceResolutionTimeout time.Duration
-	TopologyResolutionTimeout  time.Duration
+	InstrumentOpts    instrument.Options
+	HashingSeed       uint32
+	HostID            string
+	ResolutionTimeout time.Duration
```

Comment:
Need to be b/w compatible when making changes to publicly configurable fields. 
- Could you mark the old fields deprecated, 
- Add the new field,
- Error if both new and old are set, and pick the set field otherwise 

Rob had to do some of this in #890 -
 https://github.com/m3db/m3/blob/721ac9269d61247cdc8eb943f7b7b4bf9bc251c0/src/query/storage/local/config.go#L90-L117

Reply:
spoke offline, very unlikely anyone ever set these fields are they are undocumented in are YMLs so gonna make the backward incompatible change